apps/src/pythonlab/pythonHelpers/scripts.ts

```diff
@@ -1,16 +1,16 @@
+export function runValidationTests(validationFileName: string) {
+  return `
+from unittest_runner import run_validation_tests
+run_validation_tests('${validationFileName}')
+`;
+}
+
 // Find all tests in the project and run them. The test files need to follow the filePattern regex.
 // The tests need to be in the top-level folder or in a folder with an
 // __init__.py file (the file can be empty). An init file signifies that the folder is a module.
-export function getTestRunnerScript(filePattern = 'test*.py') {
+export function runStudentTests() {
   return `
-import unittest
-
-def discover_and_run(start_dir, pattern):
-    loader = unittest.TestLoader()
-    test_suite = loader.discover(start_dir, pattern)
-    runner = unittest.TextTestRunner(verbosity=2)
-    result = runner.run(test_suite)
-
-discover_and_run('.', '${filePattern}')
+from unittest_runner import run_student_tests
+run_student_tests('test*.py')
```

Comment:
👍, nice using the module to keep python code mostly in python packages, I think the future will thank you for this